policy/config/output/lsstFamAll189.yaml

```diff
@@ -0,0 +1,49 @@
+output:
```

Comment:
This seems to be a new file, right? It doesn't seem to be called anywhere else. Is that intentional?

Reply:
Yeah, I just found it to be useful to have a version of FAM that does just centers but also one that does all sensors and thought I should add it because others might want it too.